site/specs-source/voice.json

```diff
@@ -170,7 +170,7 @@
           "Calls"
         ],
         "summary": "Get Call Information",
-        "description": "Returns near-realtime metadata about the specified call.",
+        "description": "Retrieve the current state of a specific call. This information is near-realtime, so it may take a few minutes for your call to be accessible using this endpoint.<br><br>**Note**: Call information is kept for 7 days after the calls are hung up. If you attempt to retrieve information for a call that is older than 7 days, you will get an HTTP 404 response.\n\nThe `disconnectCause` for a call can be:\n- `hangup`: one party hung up the call, a [`<Hangup>`](../../bxml/verbs/hangup.md) verb was executed, or there was no more BXML to execute; it indicates that the call ended normally.\n- `busy`: the callee was busy.\n- `timeout`: the call wasn't answered before the `callTimeout` was reached.\n- `cancel`: the call was cancelled by its originator while it was ringing.\n- `rejected`: the call was rejected by the callee.\n- `callback-error`: a BXML callback couldn't be delivered to your callback server.\n- `invalid-bxml`: invalid BXML was returned in response to a callback.\n- `application-error`: an unsupported action was tried on the call, e.g. trying to play a .ogg audio.\n- `account-limit`: the account rate limits were reached.\n- `node-capacity-exceeded`: the system maximum capacity was reached.\n- `error`: some error not described in any of the other causes happened on the call.\n- `unknown`: some unknown error happened on the call.\n\nNote: this list is not exhaustive and other values can appear in the future.",
```

Comment:
```suggestion
        "description": "Retrieve the current state of a specific call. This information is near real-time, so it may take a few minutes for your call to be accessible using this endpoint.<br><br>**Note**: Call information is kept for 7 days after the calls are hung up. If you attempt to retrieve information for a call that is older than 7 days, you will get an HTTP 404 response.\n\nThe `disconnectCause` for a call can be:\n- `hangup`: one party hung up the call, a [`<Hangup>`](../../bxml/verbs/hangup.md) verb was executed, or there was no more BXML to execute; it indicates that the call ended normally.\n- `busy`: the callee was busy.\n- `timeout`: the call wasn't answered before the `callTimeout` was reached.\n- `cancel`: the call was cancelled by its originator while it was ringing.\n- `rejected`: the call was rejected by the callee.\n- `callback-error`: a BXML callback couldn't be delivered to your callback server.\n- `invalid-bxml`: invalid BXML was returned in response to a callback.\n- `application-error`: an unsupported action was tried on the call, e.g. trying to play a .ogg audio.\n- `account-limit`: the account rate limits were reached.\n- `node-capacity-exceeded`: the system maximum capacity was reached.\n- `error`: some error not described in any of the other causes happened on the call.\n- `unknown`: some unknown error happened on the call.\n\nNote: this list is not exhaustive and other values can appear in the future.",
```

Reply:
"realtime" is an adjective, but we aren't quite realtime, we're just close to realtime. So "near" is part of the adjective. When adjectives are made of multiple words, they get connected with hyphens like "near-realtime"